### PR TITLE
metrics, plan: add a metric for pseudo estimation.

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

File: plan/logical_plan_builder.go [hunk 1]
```diff
@@ -1608,6 +1608,7 @@ func (b *planBuilder) buildDataSource(tn *ast.TableName) LogicalPlan {
 				statsTbl.Count = 100
 			}
 		}
+		metrics.PseudoEstimation.Inc()
```
Comment (shenli) [inline plan/logical_plan_builder.go]: Move this line before 1610?

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@lamxTyler @winoros PTAL

File: metrics/stats.go [hunk 2]
```diff
@@ -44,10 +44,19 @@ var (
 			Help:      "Bucketed histogram of stats inaccuracy rate.",
 			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 14),
 		})
+
+	PseudoEstimation = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "statistics",
+			Name:      "pseudo_estimation",
```
Comment (coocood) [inline metrics/stats.go]: pseudo_estimation_total

**Comment by hanfei1991** [comment]
@coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**